frontend/src/components/stylists/scheduleManagementView.jsx

```diff
@@ -65,7 +65,7 @@ function ScheduleManagementView(props) {
   ) => {
     return (
       newDate.toDateString() === existingDate.toDateString() &&
-      newUserID === existingUserID
+      newUserID == existingUserID
```

Comment:
I don't think this breaks anything but it is standard in JS to use 3 = signs for comparitive operations.

Reply:
What Brian said is correct. Not using === could potentially break something.

Reply:
I understand that. Not sure where but the type of objects were giving incorrect results. Thats why the change happened. If you guys want to get to the core issue and fix it definetely give it a chance but its a fix in the meantime